native/Avalonia.Native/src/OSX/menu.mm

```diff
@@ -162,6 +162,29 @@ - (void)didSelectItem:(nullable id)sender
     }
 }
 
+HRESULT AvnAppMenuItem::SetToggleType(AvnMenuItemToggleType toggleType)
+{
+    @autoreleasepool
+    {
+        switch(toggleType)
+        {
+            case AvnMenuItemToggleType::None:
```

Comment:
I think we can fall back to NSMenuCheckmark here. So when someone sets IsChecked to true we'll still show something (dbus menu does this, but there the lack of a proper toggle type actually matters when rendering unchecked item).

Reply:
Hmmm the logic behind this was, that it would ignore IsChecked? otherwise whats the point of having ToggleTypeNone?

Reply:
ok it was decided. if ToggleType: None and user sets ischecked nothing will happen, backends ignore.